pep-8015.rst

```diff
@@ -314,15 +314,15 @@ Special Case: Ban a core developer
 
 As any other member of the Python community, the PSF Code of Conduct
 Workgroup can ban a core developer for a limited amount of time. In this
-case, the core developer immediately looses their core developer status.
+case, the core developer immediately loses their core developer status.
 
 In general, a ban is only the last resort action when all other options
-have been exausted.
+have been exhausted.
```

Comment:
That's strange. I already fixed this typo (a friend already reported it to me), but it seems like I pushed to the wrong Git repository...